### PR TITLE
implement Building design reports page and Building design structure page.

### PR DESCRIPTION
**Pull request by SallamRady**

### CODE REVIEW

File: src/pages/DesignReports/AddDesign.tsx
Comment (RustedMiind) [inline src/pages/DesignReports/AddDesign.tsx:1]: Replace States with react-hook-form
Replace style prop to sx

File: src/pages/DesignReports/components/ClassificationCardContainer/ClassificationCardContainer.tsx
Comment (RustedMiind) [inline src/pages/DesignReports/components/ClassificationCardContainer/ClassificationCardContainer.tsx:1]: Replace style prop to sx

File: src/pages/DesignReports/components/Table/Table.tsx
Comment (RustedMiind) [inline src/pages/DesignReports/components/Table/Table.tsx:1]: Replace style prop to sx

File: src/pages/DesignReports/components/classificationCard/ClassificationCard.tsx
Comment (RustedMiind) [inline src/pages/DesignReports/components/classificationCard/ClassificationCard.tsx:1]: Replace style prop to sx

**Review by RustedMiind — CHANGES_REQUESTED** [review]

**Review by RustedMiind — APPROVED** [review]

File: src/pages/DesignReports/components/ClassificationCardContainer/ClassificationCardContainer.tsx
Comment (SallamRady) [reply src/pages/DesignReports/components/ClassificationCardContainer/ClassificationCardContainer.tsx:1]: done

**Pull request closed without merging**